batch/batch/worker/worker.py

```diff
@@ -806,12 +829,20 @@ async def delete_container(self):
         if self.container_is_running():
             try:
                 log.info(f'{self} container is still running, killing crun process')
-                self.process.kill()
+                self.process.terminate()
+                self.process = None
+                await check_exec_output('crun', 'kill', '--all', self.container_name, 'SIGTERM')
             except asyncio.CancelledError:
                 raise
             except Exception:
-                log.warning('while deleting container, ignoring', exc_info=True)
-        self.process = None
+                log.exception('while deleting container', exc_info=True)
+
+            try:
```

Comment:
Did you change the structure of this from checking whether the path exists to a try/except because the path should always be there?

Reply:
The try catch here actually might not be necessary, I'm doing it a bit defensively. I previously had the check that the mount path exists because the way we currently use `delete_container` requires it be idempotent (which I'm not crazy about but that's for another time). The point of this change was moving it into the `if`,